README.md

```
@@ -39,22 +38,32 @@ BS Translator Backend is a powerful Python FastAPI service that provides advance
 # Client Configuration
 CLIENT_PORT=3000
 CLIENT_URL=http://localhost:${CLIENT_PORT}
+
+# Service URLs and auth
+HMAC_SECRET='your_hmac_secret' # generate with openssl rand 32 | base64
+DOCLING_URL='http://localhost:8004/v1'
+WHISPER_URL='http://localhost:50001/v1'
 ```
 
-> **Note:** The `HF_AUTH_TOKEN` is required for Hugging Face API access. You can create a token [here](https://huggingface.co/settings/tokens).
+> **Note:** The optimizer values are only needed when running DSPy optimization scripts. The `HF_AUTH_TOKEN` is required for Hugging Face API access and model downloads; you can create a token [here](https://huggingface.co/settings/tokens).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)